src/core/algorithms/gfd/gfd_handler.cpp

```diff
@@ -0,0 +1,62 @@
+#include "gfd_handler.h"
+
+#include <iostream>
+#include <set>
+#include <thread>
+
+#include <boost/graph/eccentricity.hpp>
+#include <boost/graph/exterior_property.hpp>
+#include <boost/graph/floyd_warshall_shortest.hpp>
+#include <boost/graph/vf2_sub_graph_iso.hpp>
+
+#include "balancer.h"
+#include "config/equal_nulls/option.h"
+#include "config/names_and_descriptions.h"
+#include "config/option_using.h"
+#include "config/tabular_data/input_table/option.h"
+#include "config/thread_number/option.h"
+
+namespace algos {
+
+GfdHandler::GfdHandler() : Algorithm({}) {
+    RegisterOptions();
+    MakeOptionsAvailable({config::names::kGfdData, config::names::kCsvPath});
+};
+
+void GfdHandler::RegisterOptions() {
+    using namespace config::names;
+    using namespace config::descriptions;
+    DESBORDANTE_OPTION_USING;
+
+    RegisterOption(config::Option{&gfd_paths_, kGfdData, kDGfdData});
+    RegisterOption(config::Option{&graph_path_, kCsvPath, kDCsvPath});

```

Comment:
The use of this option should definitely be removed. I saw your comment regarding this: 

> I had to interpret csv_path as the path to the graph, since this parameter is required for all algorithms

But this is incorrect, since for example now we are adding algorithms that expect several tables (#296) as input - these algorithms accept a vector from tables. And in general, as the desbordante develops, various new formats will be added. GFD is just this case.